app/controllers/reviews_controller.rb

```diff
@@ -10,14 +10,13 @@ def create
 
     if translation_result[:result]
       if translation_result[:typos] > 0
-        flash[:notice] = "Правильно, но при переводе совершили опечатку. Будьте внимательнее!
-                          Перевод: #{@card.original_text}, а Вы ввели: #{review_params[:user_translation]}"
+        flash[:notice] = t('views.reviews.flash_messages.user_translated_with_distance')
       else
-        flash[:notice] = "Правильно"
+        flash[:notice] = t('views.reviews.flash_messages.user_translated_correct')
       end
       redirect_to :back
     else
-      redirect_to :back, notice: "Не правильно!"
+      redirect_to :back, notice: t('views.reviews.flash_messages.user_translated_incorrect')
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.